keyFile.json

```diff
@@ -0,0 +1 @@
+{"type": "service_account", "project_id": "test", "private_key_id": "123456", "private_key": "-----BEGIN PRIVATE KEY-----test-----END PRIVATE KEY-----", "client_email": "test@project.iam.gserviceaccount.com"}
```

Comment:
Is this necessary?

Reply:
No I think it is pytest that generates it everytime, don't understand why